src/pages/mesh/advanced/logging.md

```diff
@@ -110,3 +110,19 @@ To delete the log forwarding configuration, and effectively disable log forwardi
 ```bash
 aio api-mesh:config delete log-forwarding
 ```
+
+### Get log forwarding errors
+
+To get a list of log forwarding errors, use the `aio api-mesh:config get log-forwarding errors` command.
+
+Similar to the other log forwarding commands, you can use either the `--past` flag to get logs within the past `x` minutes or the `--startTime` and `--endTime` flags to get logs from a specific time range.
```

Comment:
We don't need this. The user only needs to input `aio api-mesh:config get log-forwarding errors`